modules/administration/pages/maintenance-windows.adoc

```diff
@@ -0,0 +1,61 @@
+[[maintenance-windows]]
+= Maintenance Windows
+
+The maintenance windows feature in {productname} allows you to schedule actions to occur during a scheduled maintenance window period.
+Maintenance windows can be scheduled to suit your calendar.
+When you have created a schedule, you can select tasks that can only be performed during a scheduled maintenance window.
```

Comment:
I'm not sure I completely understand this sentence, but I think it describes the fact, that the feature restricts action execution to the maintenance windows (and does not allow executing some types of actions outside of them). I think it's important to describe this "restrictiveness" of the maintenance windows, but I find it a bit confusing.

Could we maybe merge this to the first sentence of the paragraph and rephrase to roughly something like this:

The maintenance windows feature in {productname} allows you to schedule actions to occur during a scheduled maintenance window period, and prevents you from executing actions outside of these periods.